apps/nsq_to_nsq/nsq_to_nsq.go

```diff
@@ -31,10 +31,8 @@ const (
 
 var (
 	showVersion = flag.Bool("version", false, "print version string")
-
-	topic       = flag.String("topic", "", "nsq topic")
 	channel     = flag.String("channel", "nsq_to_nsq", "nsq channel")
-	destTopic   = flag.String("destination-topic", "", "destination nsq topic")
+	singleDestinationTopic   = flag.String("single-destination-topic", "", "activate single destination nsq topic Note: default destination topic is the topic itself")
```

Comment:
I thought the "combine topics" functionality was a reasonable thing to support even though I probably wouldn't use it. In any case, I would not rename the `--destination-topic` flag, I think the existing name is fine, and it's better to not needlessly break compatibility for people who already have services configured to use it.